<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WrittenBookEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_left">
      <item>
       <widget class="QFrame" name="frame_file_options">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_0">
           <property name="text">
            <string>当前文件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_filepath">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>还没有打开任何文件呢~</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>编码格式</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmb_encoding">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>自动识别</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UTF-8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GBK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BIG5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_book_options">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chk_edit_single_page">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>编辑当前页面</string>
           </property>
           <property name="text">
            <string>编辑单页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_allow_page_split">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>允许书页末尾出现的单词被分隔到两页</string>
           </property>
           <property name="text">
            <string>允许书页分隔单词</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_force_no_wrap">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>通过在行尾强制插入换行符来禁止折行</string>
           </property>
           <property name="text">
            <string>强制不折行</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txe_text"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_right">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbtn_last_page">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_index">
           <property name="toolTip">
            <string>点击跳转到指定页面</string>
           </property>
           <property name="text">
            <string>第0卷 第0/0页 (共0页)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtn_next_page">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="gv_book_view"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="separator"/>
    <addaction name="action_open_file"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_export"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_force_unicode"/>
    <addaction name="action_jp_font"/>
   </widget>
   <widget class="QMenu" name="menu_lang">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="action_chinese"/>
    <addaction name="action_english"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
   <addaction name="menu_lang"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_open_file">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存...</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>另存为...</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionMiniMessage">
   <property name="text">
    <string>MiniMessage</string>
   </property>
  </action>
  <action name="action_force_unicode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>强制使用Unicode字体</string>
   </property>
  </action>
  <action name="action_jp_font">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>日本字形变体</string>
   </property>
  </action>
  <action name="action_allow_page_split">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>允许书页分隔单词</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_chinese">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="action_english">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
